SonarQube.TeamBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -18,6 +18,9 @@ public class WebClientDownloader : IDownloader
 
         public WebClientDownloader(string username, string password)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In member 'WebClientDownloader.WebClientDownloader(string, string)', the discrete term 'username' in parameter name 'username' should be expressed as a compound word, 'userName'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3ACompoundWordsShouldBeCasedCorrectly)